content/agent/troubleshooting.md

```diff
@@ -83,6 +83,49 @@ In the commands below, replace `<CASE_ID>` with your Datadog support case ID, if
 | Source          | `sudo ~/.datadog-agent/bin/agent flare <CASE_ID>`                       | `sudo datadog-agent flare <CASE_ID>`                  |
 | Windows         | [Consult our dedicated windows doc][9]                                  | [Consult our dedicated windows doc][10]               |
 
+## Status of an Agent check
+
+Agent checks can't be directly called from python and instead need to be called by the Agent.
+
+To test an agent check, run:
+
+| Agent version | Command                                             |
+| :------       | :----                                               |
+| v5.x          | `sudo -u dd-agent dd-agent check <CHECK_NAME>`      |
+| v6.x          | `sudo -u dd-agent datadog-agent check <CHECK_NAME>` |
+
+If you want to include rate metrics, add `--check-rate` to your command, for instance for agent v6.x run:
+
+```
+sudo -u dd-agent datadog-agent check <CHECK_NAME> --check-rate
+```
+
+If your issue continues, reach out to Support with the [help page][5] that lists the paths it installs.
```

Comment:
I don't understand this. 😕 

> […] with the help page that lists the paths it installs.

What is "it"? What are the paths? This needs to be clarified imho.